src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,164 @@ Project has the following structure:
+        title: Knot.x Starter Kit
+        description: Knotx Starter Kit
+        
+      ...
+      ```      
+ 2.  Server configuration that defines how our server behaves:
+ 
+      ```
+      ...
+      
+      servers:
+        - url: https://{domain}:{port}
+          description: The local API server
+          variables:
+            domain:
+              default: localhost
+              description: api domain
+            port:
+              enum:
+                - '8092'
+              default: '8092'
+      
+      ...
+      ```
+      Here we can see, for example, that our server will be registered on port `8092`.
+ 3.  Paths configuration that explicitly specifies our endpoints, their paths, methods, responses and much more in a clear, easily readable and declarative way:
+ 
+      ```
+      ...
+      
+      paths: 
+      
+        ...   
+             
+        /healthcheck:
+          get:
+            operationId: healthcheck-operation
+            # https://vertx.io/docs/vertx-health-check/java/
+            responses:
+              '200':
+                description: Everything is fine
+              '204':
+                description: No procedure are registered.
+              '501':
+                description: At least one procedure has reported a non-healthy stat
+              '500':
+                description: One procedure has thrown an error or has not reported a status in time
+      ...
+      ```
+      Our Starter Kit template comes with example paths and methods defined. Let's see the `/healthcheck` API. It is used to express the current state of our application in very simple terms: UP or DOWN.
+      
+      We can see that it is registered under `/healthcheck` path, it's method is a `GET` and it can have various responses depending on the state of the instance.
+      
+      There's no magic behind it. Healthcheck is handled by operation (more on that later in this tutorial) `healthcheck-operation` implemented as a handler in `HealthcheckHandlerFactory.java`. We will take a look at it later in this tutorial. Just to be brief: it simply calls our example endpoints (defined in `knotx/conf/openapi.yaml` as well). 
+      
+      This is a nice feature to have! Since we will be using Docker in this tutorial, we will tell docker to check if our instance is up and running by simply calling this endpoint and expecting a response.
+      
+      Let's take a quick peek at `docker/Dockerfile`:
+      
+      ```dockerfile
+      ... 
+      
+      HEALTHCHECK --interval=5s --timeout=2s --retries=12 \
+        CMD curl --silent --fail localhost:8092/healthcheck || exit 1
+        
+      ...
+      ```
+      Quite self-explanatory, isn't it? For more details see [this documentation](https://docs.docker.com/engine/reference/builder/#healthcheck).
+      
+Now that, we know what's going on in this file, let's define our custom endpoint.
+
+Simply add new routing configuration under `paths`:
+```hocon
```

Comment:
Since this looks like a tutorial for complete beginners, one or two sentences about HOCON might be useful. A lot of people are probably not familiar with this format.

Reply:
done